examples/java/hello_world/MainActivity.java

```diff
@@ -44,14 +45,14 @@ protected void onCreate(Bundle savedInstanceState) {
     Envoy.load(context);
 
     // Create envoy instance with config.
-    String config = null;
+    String config;
     try {
       config = loadEnvoyConfig(getBaseContext(), R.raw.config);
     } catch (RuntimeException e) {
       Log.d("MainActivity", "exception getting config.", e);
       throw new RuntimeException("Can't get config to run envoy.");
     }
-    envoy = new Envoy(context, config);
+    Envoy envoy = new Envoy(context, config);
```

Comment:
Even if it's not strictly necessary at this time, I think our apps should follow a pattern of retaining a reference to their Envoy instance, since eventually they'll need to do this anyways to utilize it. (This is a side effect of us making the decision to use an instantiated/non-singleton approach.)

Reply:
Sorry, so what I also meant by this, is can we put the L36 back (`private Envoy envoy;`) member variable. :)